sdk/storage/storage-blob/rollup.base.config.js

```diff
@@ -53,7 +53,11 @@ export function nodeConfig(test = false) {
       }),
       nodeResolve({ preferBuiltins: true }),
       cjs()
-    ]
+    ],
+    onwarn(warning, warn) {
+      if (warning.code === "CIRCULAR_DEPENDENCY") throw new Error(warning.message);
```

Comment:
nit: I'm not a huge fan of one-line if statements ;)

Reply:
Updated to blocks! :)